sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 3.0.0-preview.2 (Unreleased)
 
+### Breaking changes
+
+- Renamed the model `BoundingBox` to `FieldBoundingBox`.
+
```

Comment:
Try merging changes from master. This should be moved to 3.0.0-preview.3 since we just released a preview.2 hotfix. Alternately, we could merge this PR and get a new one out to sort stuff in the changelog.

Reply:
hah! thanks. PR updated with Master